SSZipArchive/minizip/mz_crypt_apple.c

```diff
@@ -406,121 +521,3 @@ void mz_crypt_hmac_delete(void **handle) {
     }
     *handle = NULL;
 }
-
-/***************************************************************************/
-
-#if defined(MZ_ZIP_SIGNING)
```

Comment:
With the removal of MZ_ZIP_SIGNING in minizip-ng 4, I should also update `Release-Instructions.md` regarding MZ_ZIP_SIGNING

Reply:
updated in 9a3c0342703c6a22877e75fccb5a2658422c9713